java/debugger/impl/src/com/intellij/debugger/impl/JdiHelperClassLoaderImpl.java

```diff
@@ -0,0 +1,97 @@
+// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package com.intellij.debugger.impl;
+
+import com.intellij.debugger.engine.DebugProcess;
+import com.intellij.debugger.engine.DebugProcessImpl;
+import com.intellij.debugger.engine.DebuggerUtils;
+import com.intellij.debugger.engine.evaluation.EvaluateException;
+import com.intellij.debugger.engine.evaluation.EvaluationContextImpl;
+import com.intellij.openapi.util.registry.Registry;
+import com.intellij.util.containers.ContainerUtil;
+import com.sun.jdi.*;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Objects;
+
+public class JdiHelperClassLoaderImpl implements JdiHelperClassLoader {
+  @Override
+  public @Nullable ClassType getHelperClass(Class<?> cls, EvaluationContextImpl evaluationContext,
```

Comment:
This is just a copy of what used to be in `java/debugger/impl/src/com/intellij/debugger/impl/ClassLoadingUtils.java`